spec/dummy/config/webpack/serverWebpackConfig.js

```diff
@@ -109,6 +111,31 @@ const configureServer = () => {
   // If using the React on Rails Pro node server renderer, uncomment the next line
   // serverWebpackConfig.target = 'node'
 
+  // MiniRacer specific config
+  if (process.env.EXECJS_RUNTIME === 'MiniRacer') {
```

Comment:
Where did you get these from?

Reply:
Miniracer needs a couple polyfills for some Node.js modules that are used in React 18 SSR. Here's my original post:
https://github.com/shakacode/react_on_rails/issues/1457#issuecomment-1165026717
AFAIK this isn't documented anywhere